x-pack/filebeat/module/azure/signinlogs/_meta/fields.yml

```diff
@@ -163,4 +163,51 @@
           type: keyword
           description: >
             Status
-
+        - name: authentication_requirement_policies
+          type: keyword
+          description: >
+            Set of CA policies that apply to this sign-in, each as CA: policy name, and/or MFA: Per-user.
+        - name: applied_conditional_access_policies
+          type: object
+          description: >
+            Details of the conditional access policies being applied for the sign-in.
+        - name: resource_tenant_id
+          type: keyword
+          description: >
+            The resource tenantId for B2B(business-to-business) scenarios.
+        - name: authentication_details
```

Comment:
same as above, should this be `nested`?

Reply:
Done.

(Note: this is not a new field, just was missing from fields.yml)